### ISSUE TITLE
Is multisignature.js file still in use?

### ISSUE THREAD

**Issue by a1300** [sqfasd/asch-js#19]
Dear @liangpeili 

is the [transactions/multisignature.js](https://github.com/AschPlatform/asch-js/blob/master/lib/transactions/multisignature.js) file still in use?

The [asch-js/index.js](https://github.com/AschPlatform/asch-js/blob/master/index.js) doesn't references the __transactions/multisignature.js__ file.

I couldn't find an ASCH repository that uses the functions in the __transactions/multisignature.js__ file. There are __multiSignature__ accounts in ASCH 1.4.2 but there are created through the [asch-gateway-dist](https://github.com/AschPlatform/asch-gateway-dist) repository with the function `createMultisigAddress()`

Can the [transactions/multisignature.js](https://github.com/AschPlatform/asch-js/blob/master/lib/transactions/multisignature.js) be deleted?

Thank you!
All the best
a1300

**Comment by bassjobsen** [comment]
@a1300 adding a group member requires a multisignature? https://github.com/AschPlatform/asch/blob/develop/test/integration/contract/group.test.js#L178. It uses the `multisignature` from the transaction.js now (https://github.com/AschPlatform/asch-js/blob/master/lib/transactions/transaction.js#L65). Both functions are not the same. For me it is unclear how (and which of them) i should use, see also: https://github.com/AschPlatform/asch/issues/267

**Comment by a1300** [comment]
@bassjobsen 

The test [asch/test/integration/contract/group.test](https://github.com/AschPlatform/asch/blob/develop/test/integration/contract/group.test.js#L178) is using the __asch-js__ function `transaction.createMultiSigTransaction(params)` in [asch-js/lib/transactions/transaction.js](https://github.com/AschPlatform/asch-js/blob/master/lib/transactions/transaction.js#L65). Am I wrong?

Nevertheless the [transactions/multisignature.js](https://github.com/AschPlatform/asch-js/blob/master/lib/transactions/multisignature.js) file is deprecated and should be deleted.

**Comment by bassjobsen** [comment]
@a1300 yes, you're right is uses transaction.createMultiSigTransaction(params) in asch-js/lib/transactions/transaction.js. But i'm still wondering howto use it? See https://stackoverflow.com/questions/52367434/how-can-users-sign-a-multi-signed-transaction

When removed, also https://github.com/AschPlatform/asch-core/blob/develop/test/lib/multisignatures.js can be removed  ?!

**Comment by a1300** [comment]
@bassjobsen have a look at https://github.com/AschPlatform/asch/blob/1bf70a16b91b31d8f965c3ad8279538a25212607/test/smoke/index.js#L365-L457

**Comment by bassjobsen** [comment]
@a1300 yes, thank you very much, that's useful and the tests also work. My isssue(s) are related to https://github.com/AschPlatform/asch/blob/1bf70a16b91b31d8f965c3ad8279538a25212607/test/smoke/index.js#L445-L449, howto sign if i don't know the secrets of the members? Also there seems no test in the code which checked if the transaction is signed or not.
Everyone who pays the fee can add / or remove members from a group, even when you are not a member of the group.

**Comment by a1300** [comment]
@bassjobsen let the discussion continue on [gitter.im](https://gitter.im/asch-development/Lobby)
